ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/internal/trailingcomma/TrailingCommaExtensions.kt

```diff
@@ -0,0 +1,140 @@
+package com.pinterest.ktlint.ruleset.standard.internal.trailingcomma
```

Comment:
I would strongely prefer not to have a shared helper for the two trailing comma rules. The emit and autocorrect should be kept in the Rule class. Also I do expect that most of the logic is used by only one of the two classes anyway.